docs/reference/content/bson/extended-json.md

```diff
@@ -7,23 +7,30 @@ title = "Extended JSON"
   pre = "<i class='fa'></i>"
 +++
 
-## MongoDB Extended JSON
-
-As discussed earlier, the Java driver supports reading and writing BSON documents represented as  
-[MongoDB Extended JSON](http://docs.mongodb.org/manual/reference/mongodb-extended-json/).  Both variants are supported: 
-
-- Strict Mode: representations of BSON types that conform to the [JSON RFC](http://www.json.org/). This is the 
-format that [mongoexport](http://docs.mongodb.org/manual/reference/program/mongoexport/) produces and 
-[mongoimport](http://docs.mongodb.org/manual/reference/program/mongoimport/) consumes.
+## JSON
+
+As discussed earlier, the Java driver supports reading and writing BSON documents represented as JSON values. The driver supports four 
+standard variants:   
+
+- Extended Mode: Canonical representation that avoids any loss of BSON type information. See the 
+[Extended JSON specification](https://github.com/mongodb/specifications/blob/master/source/extended-json.rst) for a description of this 
+mode.
+- Relaxed Mode:  Relaxed representation that loses type information for BSON numeric types and uses a more human-readable representation
+of BSON dates. See the 
+[Extended JSON specification](https://github.com/mongodb/specifications/blob/master/source/extended-json.rst) for a description of this 
+mode.
+default mode when writing JSON.
```

Comment:
I'm pleased we're defaulting to this, but we will need to include the change in the upgrade notes - as this could catch some users out.